### PR TITLE
Add type to IS queries

### PR DESCRIPTION
**Pull request by gordonwang0**
[This change](https://github.com/Azure/iot-identity-service/commit/1ed1cb1a6144c5562c9a3bb00a5d7b5fc66b8f04) to aziot updates the Identity Service APIs to require a `type` parameter.

This PR updates IS calls to include the type parameter.

### CODE REVIEW

File: edgelet/identity-client/src/client.rs [hunk 1]
```diff
@@ -120,7 +120,7 @@ impl IdentityClient {
     ) -> Box<dyn Future<Item = aziot_identity_common::Identity, Error = Error> + Send> {
         let client = self.client.clone();
         let uri = format!(
-            "/identities/modules/{}?api-version={}",
+            "/identities/modules/{}?api-version={}&type=aziot&type=aziot",
```
Comment (massand) [inline edgelet/identity-client/src/client.rs:123]: typo

File: edgelet/management/src/apis/identity_api.rs [hunk 2]
```diff
@@ -63,6 +63,7 @@ where
 
         let query = ::url::form_urlencoded::Serializer::new(String::new())
             .append_pair("api-version", &api_version.to_string())
+            .append_pair("type", "aziot")
```
Comment (massand) [inline edgelet/management/src/apis/identity_api.rs:66]: These are code-gen and shouldn't be changed

File: edgelet/identity-client/src/client.rs (hunk 1 above)
Comment (gordonwang0) [reply edgelet/identity-client/src/client.rs]: Fixed

File: edgelet/management/src/apis/identity_api.rs (hunk 2 above)
Comment (gordonwang0) [reply edgelet/management/src/apis/identity_api.rs]: Fixed

**Review by massand — APPROVED** [review]

**Pull request merged**